transformer_engine/pytorch/module/layernorm_linear.py

```diff
@@ -1292,13 +1311,29 @@ def forward(
             else:
                 bias_tensor = getattr(self, self.bias_names[0])  # Unused
 
+            debug = TEDebugState.debug_enabled
```

Comment:
Could we move this to the beginning of the function? You use the state there as well.